decidim-proposals/spec/system/amendable/amend_proposal_spec.rb

```diff
@@ -208,8 +208,8 @@
 
           before do
             expect(page).to have_content("Log in")
```

Comment:
I think this is why the specs are failing

```suggestion
            expect(page).to have_content("Log in")
            switch_to_host(component.organization.host)
```

Reply:
Lets try!

Reply:
Fixed both issues within `proposals`. See latest commits.